docs/styles/config/vocabularies/icalendar/accept.txt

```diff
@@ -104,3 +104,4 @@ vCard
 (?i)Wieser
 (?i)Zeidler
 Zepyoor
+(?i)Penner
```

Comment:
Please move this into an appropriate alphabetic sort order. You also don't need to prefix it with the case-insensitive regular expression flag, since you don't use it lowercased anywhere in the docs.

```suggestion
Penner
```

Also is `Lillian` required to pass the Vale spellchecker warning?